sycl/source/detail/platform_impl.cpp

```diff
@@ -58,16 +58,26 @@ struct DevDescT {
 
   const char *devDriverVer = nullptr;
   int devDriverVerSize = 0;
+
+  const char *pltName = nullptr;
+  int pltNameSize = 0;
```

Comment:
Can you replace "plt" with "platform" everywhere? Unlike "dev" it is not really obvious.

Reply:
Done.